src/runtime/policy/tests/policy-parser-test.ts

```diff
@@ -57,6 +72,16 @@ describe('policy parser', () => {
     });
   });
 
+  it('parses a policy with annotations', () => {
+    const policy = parsePolicy(`@noParam
+@oneParam('aaa')
```

Comment:
Is indentation wrong here?

Reply:
It's not wrong, just weird... There's no indent before either `@` symbol. I wish JS had a handy stripIndent function...